cdc/scheduler.go

```diff
@@ -357,23 +58,21 @@ func realRunProcessor(
 	processor, err := NewProcessor(ctx, pdEndpoints, info, sink, changefeedID, captureID, checkpointTs)
 	if err != nil {
 		cancel()
-		return err
+		return nil, err
 	}
 	log.Info("start to run processor", zap.String("changefeed id", changefeedID))
 
-	if cb != nil {
-		cb.OnRunProcessor(processor)
-	}
-
 	processor.Run(ctx, errCh)
 
 	go func() {
 		err := <-errCh
-		if cb != nil {
-			cb.OnStopProcessor(processor, err)
-		}
+		log.Error("error on running processor",
```

Comment:
If the `errors.Cause(err)` is `context.Cancel`, we should ignore it

Reply:
Why should be ignored?